etc/example_submission_config.yaml

```diff
@@ -27,6 +27,12 @@ ena:
   username: user
   password: pass
 
+postgres:
```

Comment:
Sorry, this part seems to have completely escaped my attention during the previous review! In order to not scatter credentials across multiple files, we use a helper methods for [Postgres](https://github.com/EBIvariation/eva-common-pyutils/blob/803b4140b9936df7e9973f05e0faadde1ad00c6d/ebi_eva_common_pyutils/config_utils.py#L38) and [MongoDB](https://github.com/EBIvariation/eva-common-pyutils/blob/803b4140b9936df7e9973f05e0faadde1ad00c6d/ebi_eva_common_pyutils/config_utils.py#L47) and pass a secrets configuration file as parameter. I will message you more details in Slack.